javascript/javascript1/week1/java.js

```diff
@@ -0,0 +1,103 @@
+// Age-ify (A future age calculator)
+const yearOfMyBirth = 1986;
+const yearFuture = 2035;
+const age = yearFuture - yearOfMyBirth;
+console.log("My Firend will be " + age + " years old in " + yearFuture);
+// Goodboy-Oldboy (A dog age calculator)
+const dogyearOfbirth = 2020;
+const dogyearFuture = 2028;
+let dogage = dogyearFuture - dogyearOfbirth;
+let typeofyaer = "human";
+console.log ("My Firend's Dog will "   +   dogage + " years ");
+
+//  result in dog years or human years
+const shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears) {dogage = 20
+typeofyaer = "dog"}
+console.log (shouldShowResultInDogYears);
+
+
+// Housey pricey (A house price estimator)
+// Peter home
+const Peter = [
+    
+        houseWide = 8,
+        houseDeep = 10,
+        houseHigh = 10,
+        gardenSizeInM2 =  100,
+        housecost = 2500000,
+]
+let volumeInMetersPeter ;
+volumeInMeterspeter = houseWide * houseDeep * houseHigh;
+housePricePeter = volumeInMeterspeter * 2.5 * 1000 + gardenSizeInM2 * 300;
+let AnalazPeter;
+const Max = "too Much";
+const min = "Too low";
```

Comment:
I'd recommend you use at least 2 word variable names and consistent case. The first will help you avoid using [reserved variable and function names](https://www.w3schools.com/js/js_reserved.asp). Using a cConsistent case, would make it easier to identify if something is intended to be a variable or a function, or Object (those two are written with uppercase).
```suggestion
const messageHighPrice = "Too high price";
const messageLowPrice = "Good price";
```